net/wireshark3/Portfile

```
@@ -139,6 +139,10 @@ variant python37 description {Use python37 during build} {
     depends_build-append    port:python37
 }
 
+variant python38 description {Use python38 during build} {
```

```suggestion
variant python38 conflicts python37 description {Use python38 during build} {
```

also should be applied to other pythonxx variants

---

ok, I get it, it actually doesn't cause a conflict